### DISCUSSION TITLE
Email avisando cliente abrindo OS

### DISCUSSION THREAD

**Discussion by yasron** [RamonSilva20/mapos#1592] [Ideas]
Gostaria de saber se já tem, ou teria algum programador para criar um comando para receber um e-mail quando o cliente se cadastra e gera um os.
 Seria uma boa implementação eu acho para o sistema que já é muito bom.

**Comment by Pr3d4dor** [comment]
@yasron Atualmente quando uma OS é atualizada, é enviado email para o responsável pela mesma.

**Reply by yasron** [reply]
@Pr3d4dor sim certo, isso é muito bom e muito útil, mais minha ideia é deixar o link de acesso do cliente disponível, para que ele possa abrir uma o.s, como se abrisse um ticket, ele se cadastrando na área do cliente eu receber um e-mail informando que o cliente se cadastrou e assim eu podendo ver se abriu uma o.s, conseguiu me entender?
